src/renderer/account/components/AccountPanel/Breakdown/BreakdownChart/BreakdownChart.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import classNames from 'classnames';
+import { ResponsiveContainer, PieChart, Pie, Cell, Tooltip } from 'recharts';
+import { string } from 'prop-types';
+import { times, reduce } from 'lodash';
+
+import formatCurrency from 'account/util/formatCurrency';
+
+import SectionLabel from '../SectionLabel';
+import chartDataShape from '../../../../shapes/chartDataShape';
+import styles from './BreakdownChart.scss';
+
+const COLORS = ['#5ebb46', '#b5d433', '#0b99e3'];
+
+function reduceSum(data) {
+  return reduce(data, (sum, datum) => sum + datum.value, 0);
+}
+
+export default class BreakdownChart extends React.PureComponent {
+  static propTypes = {
+    className: string,
+    data: chartDataShape.isRequired,
+    currency: string.isRequired
+  };
+
+  static defaultProps = {
+    className: null
+  };
+
+  render() {
+    const data = this.getData();
+
+    return (
+      <ResponsiveContainer width="100%" className={classNames(styles.breakdownChart, this.props.className)}>
+        <PieChart>
+          <Pie
+            data={data}
+            dataKey="value"
+            nameKey="symbol"
+            innerRadius="50%"
+            outerRadius="65%"
+            startAngle={90}
+            endAngle={-270}
+            label={SectionLabel}
+            labelLine={false}
+            isAnimationActive={false}
+          >
+            {times(data.length, (index) => (
+              <Cell
+                key={`cell-${index}`}
+                fill={COLORS[index % COLORS.length]}
```

Comment:
the chart only has 3 colors available for 3 different tokens? Or how should I interpret this :P

Reply:
We should up to 3 regions.  After the first 2, we group everything else under the label "OTHER".  This helps to improve the readability of the chart so there aren't a bunch of overlapping labels that represent 1% of holdings all over the place.